lib/src/blocs/filtered_todos/filtered_todos_bloc.dart

```diff
@@ -0,0 +1,91 @@
+import 'dart:async';
```

Comment:
why do filtered todo's have their own Bloc 🤔 

Isn't the idea that we have one TodoBloc where things like filtering can be done from that one bloc

Reply:
😅 Let me move it into one bloc

Reply:
I just remembered why it was there. In case a person wants more filter controls,  it was better to leave the filtering to its own dedicated bloc. They also did the same on the flutter bloc site.

Reply:
fixed it